boofuzz/fuzzable.py

```diff
@@ -50,6 +50,8 @@ def __init__(self, name=None, default_value=None, fuzzable=True, fuzz_values=Non
             Fuzzable.name_counter += 1
             self._name = "{0}{1}".format(type(self).__name__, Fuzzable.name_counter)
 
+        self.index_mutation = 0
```

Comment:
What do you think about `mutation_index`?
```suggestion
        self.mutation_index = 0
```